.gitignore

```diff
@@ -5,3 +5,4 @@ docs/_build
 build
 data
 dist/
+.idea/
```

Comment:
Could you add this to your [global .gitignore file](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore) instead of here? We try not to have IDE specific patterns here.

Reply:
Ok, no problem!